linkcheck/configuration/__init__.py

```diff
@@ -303,10 +303,12 @@ def sanitize_loginurl(self):
         url = self["loginurl"]
         disable = False
         if not self["loginpasswordfield"]:
-            log.warn(LOG_CHECK, _("no CGI password fieldname given for login URL."))
+            log.warn(LOG_CHECK,
+                     _("no password input name attribute given for login URL."))
             disable = True
         if not self["loginuserfield"]:
-            log.warn(LOG_CHECK, _("no CGI user fieldname given for login URL."))
+            log.warn(LOG_CHECK,
+                     _("no username input name attribute given for login URL."))
             disable = True
```

Comment:
Hey, didn't we have a bug about supporting login forms with just a password field, without username?

A few dozen lines down from this you have `if disable: self["loginurl"] = None`.

Reply:
Interesting. Possibly we can just delete the individual checks. Some thought and testing required.